qos/cometbft/response_status.go

```diff
@@ -76,6 +83,12 @@ type responseToStatus struct {
 	// jsonRPCResponse stores the JSON-RPC response parsed from an endpoint's response bytes.
 	jsonRPCResponse jsonrpc.Response
 
+	// chainID stores the chain ID of the endpoint.
+	chainID string
+
+	// synced indicates if the endpoint is synced to the network.
+	synced bool
```

Comment:
Why not "catching_up"?

Reply:
Yesterday I thought `synced` was clearer; now I disagree with myself from yesterday. 😅 I've changed to catching up.